.github/workflows/assign-codeowners.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Assign code owners'
+on:
+  pull_request:
```

Comment:
See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/workflows/auto-assign-owners.yml#L3, I have not thought about any security issues, so please do that :)

Reply:
Yea that'll definitely give PRs from forks write access. There are definitely some security risks according to GitHub (I'll post the article tomorrow).

I'll do some experimenting to see if that target is necessary still once the workflow is checked in, but I don't think it is.

Reply:
Here is the article talking about pull_request_target: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/.  Best practice with this event is to not checkout code.